modules/nlu/src/views/lite/intentEditor/IntentHint.tsx

```diff
@@ -9,17 +10,24 @@ import style from './style.scss'
 interface Props {
   intent: sdk.NLU.IntentDefinition
   contentLang: string
-  mlRecommendations: NluMlRecommendations
+  axios: AxiosInstance
 }
 
-const IntentHint: SFC<Props> = props => {
+const fetchRecommendations = async (axios: AxiosInstance): Promise<NluMlRecommendations> => {
+  return axios.get('/mod/nlu/ml-recommendations').then(({ data }) => data)
```

Comment:
Could you move that to the api, so we can remove axios properties here and its parents?

Reply:
another pr